platform/android/export/export.cpp

```diff
@@ -778,6 +778,87 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		}
 	}
 
+	void _write_tmp_manifest(const Ref<EditorExportPreset> &p_preset, bool p_give_internet, bool p_debug) {
+		String manifest_header =
+				"<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
+				"<manifest xmlns:android=\"http://schemas.android.com/apk/res/android\"\n"
+				"    xmlns:tools=\"http://schemas.android.com/tools\">\n";
+
+		String manifest_screen_sizes = _get_screen_sizes_tag(p_preset);
+		String manifest_gles = _get_gles_tag();
+
+		Vector<String> perms;
+		_get_permissions(p_preset, p_give_internet, perms);
+		String manifest_permission;
+		for (int i = 0; i < perms.size(); i++) {
+			manifest_permission = manifest_permission.insert(0, "\t<uses-permission android:name=\"" + perms.get(i) + "\"/>\n");
+		}
+
+		String xr_metadata_text = "";
+		String fa_metadata_text = "";
+		String head_tracking = "";
+		String hand_tracking = "";
+		int xr_mode_index = p_preset->get("xr_features/xr_mode");
+		if (xr_mode_index == 1 /* XRMode.OVR */) {
+			String focus_awareness = bool_to_string(p_preset->get("xr_features/focus_awareness"));
+			xr_metadata_text = "<meta-data android:name=\"com.samsung.android.vr.application.mode\" android:value=\"vr_only\" />\n";
+			fa_metadata_text = "<meta-data android:name=\"com.oculus.vr.focusaware\" android:value=\"" + focus_awareness + "\"/>\n";
+			int dof_index = p_preset->get("xr_features/degrees_of_freedom"); // 0: none, 1: 3dof and 6dof, 2: 6dof
+			if (dof_index == 2) {
+				head_tracking = "<uses-feature android:name=\"android.hardware.vr.headtracking\" android:required=\"true\" android:version=\"1\"/>\n";
+			} else if (dof_index > 0) {
+				head_tracking = "<uses-feature android:name=\"android.hardware.vr.headtracking\" android:required=\"false\" android:version=\"1\"/>\n";
+			}
+			int hand_tracking_index = p_preset->get("xr_features/hand_tracking"); // 0: none, 1: optional, 2: required
+			if (hand_tracking_index == 2) {
+				hand_tracking = "<uses-feature android:name=\"oculus.software.handtracking\" android:required=\"true\"/>\n";
+			} else if (hand_tracking_index > 0) {
+				hand_tracking = "<uses-feature android:name=\"oculus.software.handtracking\" android:required=\"false\"/>\n";
+			}
+		}
+
+		String package_name = p_preset->get("package/unique_name");
+		String instrumentation = "<instrumentation\n"
```

Comment:
I tried moving the instrumentation class to the `app` module, but then it didn't build.